api/graphql/schema.graphql

```diff
@@ -21,7 +21,7 @@ type Query {
 
   # Query for PersonConnections
   connections(
-    # Number of PersonConnections to load
+    # Int of PersonConnections to load
```

Comment:
This comment seems more confusing. It is meant to say that this is the number of person connections to load. the type is already visible in the graphiql viewer so doesnt need to be repeated in the comment